pkg/metrics/metrics.go

```diff
@@ -179,6 +181,22 @@ func NewServerMetrics() *ServerMetrics {
 				},
 				[]string{scheduleLabel},
 			),
+			backupItemsTotalGauge: prometheus.NewGaugeVec(
+				prometheus.GaugeOpts{
+					Namespace: metricNamespace,
+					Name:      backupItemsTotalGauge,
+					Help:      "Total number of items backed up",
+				},
+				[]string{scheduleLabel},
+			),
+			backupItemsErrorsGauge: prometheus.NewGaugeVec(
+				prometheus.GaugeOpts{
+					Namespace: metricNamespace,
+					Name:      backupItemsErrorsGauge,
+					Help:      "Total number of items which could not be backed up",
```

Comment:
I wonder whether it would be better to just say something like "Total number of errors encountered during backup" -- I don't know that the number of errors is guaranteed to be equal to the number of items that failed to back up. Errors is the error message count rather than explicitly an item count. It's possible that the number of errors and the number of items aren't identical (i.e. an error message generated separate from an item backup failure, or if more than one error log message is generated for certain failure scenarios).

Reply:
Totally fine for me to change the text, sure. 

```suggestion
					Help:      "Total number of errors encountered during backup",
```